apps/frontend/src/pages/people/[username].tsx

```diff
@@ -186,7 +216,8 @@ const UserProfilePage: React.FC = () => {
       }
 
       setConnection(user.connections[connection.user.username]);
-      handleCloseTapModal();
+      setShowCommentModal(false);
+      await storage.deleteSavedTapInfo();
```

Comment:
Why is the tap info being deleted?

Reply:
this is just saved tap info, its the temporary tap thats saved from when you tap so the info is surfaced for registration, etc.